tools/cellxgene_census_builder/src/cellxgene_census_builder/build_soma/globals.py

```diff
@@ -58,126 +72,118 @@
-# https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md
+# https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/4.0.0/schema.md
 #
 # NOTE: a few additional columns are added (they are not defined in the CXG schema),
 # eg., dataset_id, tissue_general, etc.
-CXG_OBS_TERM_COLUMNS = {
-    # Columns pulled from the CXG H5AD
-    "assay": pa.large_string(),
-    "assay_ontology_term_id": pa.large_string(),
-    "cell_type": pa.large_string(),
-    "cell_type_ontology_term_id": pa.large_string(),
-    "development_stage": pa.large_string(),
-    "development_stage_ontology_term_id": pa.large_string(),
-    "disease": pa.large_string(),
-    "disease_ontology_term_id": pa.large_string(),
-    "donor_id": pa.large_string(),
-    "is_primary_data": pa.bool_(),
-    "observation_joinid": pa.large_string(),
-    "self_reported_ethnicity": pa.large_string(),
-    "self_reported_ethnicity_ontology_term_id": pa.large_string(),
-    "sex": pa.large_string(),
-    "sex_ontology_term_id": pa.large_string(),
-    "suspension_type": pa.large_string(),
-    "tissue": pa.large_string(),
-    "tissue_ontology_term_id": pa.large_string(),
-    "tissue_type": pa.large_string(),
-}
-CENSUS_OBS_STATS_COLUMNS = {
-    # Columns computed during the Census build and written to the Census obs dataframe.
-    "raw_sum": pa.float64(),
-    "nnz": pa.int64(),
-    "raw_mean_nnz": pa.float64(),
-    "raw_variance_nnz": pa.float64(),
-    "n_measured_vars": pa.int64(),
-}
-CENSUS_OBS_TERM_COLUMNS = {
-    # Columns written to the Census obs dataframe.
-    "soma_joinid": pa.int64(),
-    "dataset_id": pa.large_string(),
-    **CXG_OBS_TERM_COLUMNS,
-    "tissue_general": pa.large_string(),
-    "tissue_general_ontology_term_id": pa.large_string(),
-    **CENSUS_OBS_STATS_COLUMNS,
-}
-
-"""
-Materialization of obs/var schema in TileDB is tuned with the following, largely informed by empirical testing:
-* string columns with repeating labels use DictionaryFilter, which efficiently encodes these highly repetative strings.
-  Columns with non-repetative values (e.g., var.feature_id) are NOT dictionary coded.
-* obs/var DataFrame show significant improvement in on-disk size and read performance at high Zstd compression level,
-  making it worth the extra build/write compute time. Level set to highest value that does not require additional
-  resources at decompression (nb. 20+ requres additional memory).
-* int64 index columns (soma_joinid, soma_dim_0, etc) empirically show wins from ByteShuffle followed by Zstd.
-  First dimension of axis dataframes are always monotonically increasing, and also beneit from DoubleDelta.
-* Benchmarking X slicing (using lung demo notebook) used to tune X[raw]. Read / query performance did not benefit from
-  higher Zstd compression beyond level=5, so the level was not increased further (and level=5 is still reasonable for
-  writes)
-"""
```

Comment:
I appreciated these notes, but also understand if they're now sufficiently outdated to not bother keeping (and updating).